utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/sink/UniversalKafkaSinkFactory.scala

```diff
@@ -4,31 +4,34 @@ import cats.data.NonEmptyList
 import io.confluent.kafka.schemaregistry.ParsedSchema
 import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.CustomNodeError
 import pl.touk.nussknacker.engine.api.context.ValidationContext
-import pl.touk.nussknacker.engine.api.context.transformation.{DefinedEagerParameter, NodeDependencyValue}
+import pl.touk.nussknacker.engine.api.context.transformation.{BaseDefinedParameter, DefinedEagerParameter, NodeDependencyValue}
 import pl.touk.nussknacker.engine.api.definition._
 import pl.touk.nussknacker.engine.api.process.{ProcessObjectDependencies, Sink, SinkFactory}
 import pl.touk.nussknacker.engine.api.{LazyParameter, MetaData, NodeId}
-import pl.touk.nussknacker.engine.avro.KafkaAvroBaseComponentTransformer.{SchemaVersionParamName, SinkKeyParamName}
+import pl.touk.nussknacker.engine.avro.KafkaAvroBaseComponentTransformer._
 import pl.touk.nussknacker.engine.avro.encode.ValidationMode
 import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.UniversalSchemaSupport
 import pl.touk.nussknacker.engine.avro.schemaregistry.{SchemaBasedSerdeProvider, SchemaRegistryClientFactory}
-import pl.touk.nussknacker.engine.avro.sink.UniversalKafkaSinkFactory.TransformationState
-import pl.touk.nussknacker.engine.avro.{KafkaAvroBaseComponentTransformer, KafkaUniversalComponentTransformer, RuntimeSchemaData, SchemaDeterminerErrorHandler}
+import pl.touk.nussknacker.engine.avro.sink.UniversalKafkaSinkFactory.{RawEditorParamName, TransformationState}
+import pl.touk.nussknacker.engine.avro.{KafkaUniversalComponentTransformer, RuntimeSchemaData, SchemaDeterminerErrorHandler}
+import pl.touk.nussknacker.engine.util.output.OutputValidatorErrorsConverter
 import pl.touk.nussknacker.engine.util.sinkvalue.SinkValue
-import pl.touk.nussknacker.engine.util.sinkvalue.SinkValueData.SinkValueParameter
+import pl.touk.nussknacker.engine.util.sinkvalue.SinkValueData.{SinkSingleValueParameter, SinkValueParameter}
 
 /**
-  * This is universal kafka sink - it will handle both avro and json
-  * TODO: Move it to some other module when json schema handling will be available
-  */
+ * This is universal kafka sink - it will handle both avro and json
+ * TODO: Move it to some other module when json schema handling will be available
+ */
 object UniversalKafkaSinkFactory {
 
+  val RawEditorParamName = "Raw editor"
```

Comment:
I'm not sure if users will know what is it "editor". AFAIK we haven't introduced this internal anywhere yet. Maybe "Raw output value"? (I'm not sure about this name)

Reply:
Or let's wait for users feadback